systemd/crc-needs-tap.sh

```
@@ -0,0 +1,57 @@
+    echo "Running on AWS. Don't need tap0."
+    exit "$DONT_NEED_TAP"
+fi
+
+virt="$(systemd-detect-virt || true)"
+
+if "${virt}" == "apple" ; then
+    echo "Running with 'virt' virtualization. Don't need tap0."
+    exit "$DONT_NEED_TAP"
+fi
+
+if "${virt}" =~ ^(kvm|microsoft)$ ; then
```

Shellcheck (0.11.0)

[error] Couldn't parse this if expression. Fix to allow more checks.

(SC1073)